book/src/SUMMARY.md

```diff
@@ -12,6 +12,8 @@
 - [Development](development/README.md)
     - [Basics](development/basics.md)
     - [Adding Lints](development/adding_lints.md)
+    - [Define Lints](development/define_lints.md)
```

Comment:
A `define_lints.md` file is created with no contents, maybe a leftover from other PRs?